spec.html

```diff
@@ -8878,11 +8883,12 @@ <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _protot
         1. If _realm_ is not present, set _realm_ to the current Realm Record.
         1. Assert: _realm_ is a Realm Record.
         1. If _prototype_ is not present, set _prototype_ to _realm_.[[Intrinsics]].[[%Function.prototype%]].
-        1. Let _func_ be a new built-in function object that when called performs the action described by _steps_. The new function object has internal slots whose names are the elements of _internalSlotsList_.
+        1. Let _func_ be a new built-in function object that when called performs the action described by _steps_. The new function object has internal slots whose names are the elements of _internalSlotsList_, and an [[InitialName]] internal slot.
```

Comment:
Commenting here as a reminder. Don't need to be acted on for this PR.

This object creation is weird. Built-in objects can either be ordinary function objects or exotic. This AO tries to be generic for both, with loose language about the internal slots in prose. We should clean this up to be similar to other object allocation sites and, and perhaps also split the exotic version from the ordinary versoin.